src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import lotto.util.NumberGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.IntStream;
+
+import static lotto.constant.ErrorMessage.DUPLICATE_LOTTO_NUMBER;
+import static lotto.constant.ErrorMessage.INVALID_LOTTO_NUMBER_COUNT;
+
+public class Lotto {
+    public static final int PRICE = 1000;
```

Comment:
사소하지만 세자리수 마다 `_`를 명시적으로 선언할 수 있어요. 
가독성을 위해서라도 가능하면 `_`로 자릿수를 한눈에 구분하기 쉽게 해주시면 좋을 것 같아요. 

```suggestion
    public static final int PRICE = 1_000;
```

Reply:
이보다 더 길이가 긴 숫자의 경우, 제가 불편함을 느껴 `_`를 사용했지만, 여기선 사용하지 않았네요.
앞으로 좀 더 의식해보도록 하겠습니다!